### PR TITLE
Feature texteare and multiline input

### PR DESCRIPTION
**Pull request by pochretien**

### CODE REVIEW

**Comment by pochretien** [comment]
image

**Comment by pochretien** [comment]
image

**Comment by JPLachance** [comment]
How does the textarea look with a plceholder?

File: docs/_components/form_controls/19_multiline-input.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                <label>Multiline one value entered</label>
+                <div class="input-actions">
+                    <button class="js-delete-value-button" type="button">
+                        <i class="delete-action" title="Delete this entry"></i>
+                    </button>
+                </div>
+            </li>
+            <li class="input-wrapper">
+                <input type="text" class="js-value-input" value="Second multiline value" required>
+                <label>Multiline one value entered</label>
```
Comment (JPLachance) [inline docs/_components/form_controls/19_multiline-input.html]: I'm not sure this one should have a label.

File: scss/controls/multiline-field.scss [hunk 2]
```diff
@@ -1,17 +1,98 @@
 .multiline-field {
-  li {
-    margin-bottom: 30px;
+  ul {
```
Comment (JPLachance) [inline scss/controls/multiline-field.scss]: ul il { .. }
I prefer declare those on a single line.

**Comment by JPLachance** [comment]
Good job :)

**Pull request merged**